app/views/taxons/show_a.html.erb

```diff
@@ -2,32 +2,32 @@
 <% content_for :is_taxon_with_subtopics, presented_taxon.show_subtopic_grid? %>
 <%=
   render(
-    partial: 'common',
-    locals: {
-      presented_taxon: presented_taxon
-    }
+      partial: 'common',
+      locals: {
+          presented_taxon: presented_taxon
+      }
   )
 %>
 
 <div class="full-page-width-wrapper">
   <% presented_taxon.sections.each do |section| %>
     <% if section[:show_section] %>
-    <div class="taxon-page__section-group">
-      <div class="grid-row">
-        <div class="column-two-thirds">
-          <%= render "govuk_publishing_components/components/heading", {
-            text: t(section[:id], scope: :content_purpose_supergroup, default: section[:title]),
-            heading_level: 2,
-            margin_bottom: 2
-          } %>
+      <div class="taxon-page__section-group">
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <%= render "govuk_publishing_components/components/heading", {
+                text: t(section[:id], scope: :content_purpose_supergroup, default: section[:title]),
+                heading_level: 2,
+                margin_bottom: 2
+            } %>
+          </div>
         </div>
+        <%= render(
+                partial: section[:partial_template],
```

Comment:
Same comment here about indenting, I think it's more easily readable if it's only indented one tab.